npm/private/transitive_closure.bzl

```diff
@@ -103,13 +89,6 @@ def translate_to_transitive_closure(importers, packages, prod = False, dev = Fal
         deps = prod_deps | opt_deps
         all_deps = prod_deps | dev_deps | opt_deps
 
-        # Package versions mapped to alternate versions
```

Comment:
This was overwriting far too much because it was only based on name causing things like https://github.com/aspect-build/rules_js/issues/2300

Reply:
removed in a prefactor now